src/lib/flexbox/api/base.ts

```diff
@@ -127,7 +135,7 @@ export abstract class BaseFxDirective implements OnDestroy, OnChanges {
     let value = this._lookupStyle(element, 'display');
 
     // Note: 'inline' is the default of all elements, unless UA stylesheet overrides
-    return value ? value.trim() : 'inline';
+    return value ? value.trim() : ((element.nodeType === 1) ? 'block' : 'inline');
```

Comment:
Not a huge of nested ternaries. Consider breaking it out into an `if` and a ternary? E.g.

```
if (value) {
  return value;
}

return ((element.nodeType === 1) ? 'block' : 'inline');
```